src/main/java/it/gov/pagopa/payhub/activities/mapper/ingestionflow/debtposition/InstallmentSynchronizeMapper.java

```diff
@@ -0,0 +1,109 @@
+package it.gov.pagopa.payhub.activities.mapper.ingestionflow.debtposition;
+
+import it.gov.pagopa.payhub.activities.dto.ingestion.debtposition.InstallmentIngestionFlowFileDTO;
+import it.gov.pagopa.pu.debtposition.dto.generated.InstallmentSynchronizeDTO;
+import it.gov.pagopa.pu.debtposition.dto.generated.TransferSynchronizeDTO;
+import org.apache.commons.collections4.MultiValuedMap;
+import org.springframework.context.annotation.Lazy;
+import org.springframework.stereotype.Service;
+
+import java.math.BigDecimal;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.IntStream;
+
+import static it.gov.pagopa.payhub.activities.util.Utilities.bigDecimalEuroToLongCentsAmount;
+
+@Service
+@Lazy
+public class InstallmentSynchronizeMapper {
+
+    public InstallmentSynchronizeDTO map(InstallmentIngestionFlowFileDTO installmentIngestionFlowFileDTO,
+                                         Long ingestionFlowFileId,
+                                         Long organizationId) {
+        return InstallmentSynchronizeDTO.builder()
+                .ingestionFlowFileId(ingestionFlowFileId)
+                .ingestionFlowFileLineNumber(installmentIngestionFlowFileDTO.getIngestionFlowFileLineNumber())
```

Comment:
is this field really updated?

Reply:
do we really need this field? could it be ambiguous to have it stored here?.
 Where actually you are setting it, you are invoking the code that should then use it, maybe it could be clearer to remove from this DTO and just pass the value to invoked method `debtPositionService.installmentSynchronize(...)`